### ISSUE TITLE
CI: K8sUpdates Tests upgrade and downgrade from a Cilium stable image to master

### ISSUE THREAD

**Issue by nbusseneau** [cilium/cilium#19870]
### Test Name

```shell
K8sUpdates Tests upgrade and downgrade from a Cilium stable image to master
```

### Failure Output

```shell
Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: Service "hubble-peer" in namespace "kube-system" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "cilium-preflight": current value is "cilium"
```

### Stack Trace

```shell
/home/jenkins/workspace/cilium-v1.11-k8s-1.19-kernel-4.9/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:527
Timed out after 30.001s.
Unable to deploy preflight manifest
Expected command: helm install cilium-preflight /home/jenkins/workspace/cilium-v1.11-k8s-1.19-kernel-4.9/src/github.com/cilium/cilium/install/kubernetes/cilium --version=1.10.90 --namespace=kube-system  --set hubble.relay.image.tag=b0d3140aa079b31d2f3889b0f695cf7add393e68  --set hubble.eventBufferCapacity=65535  --set pprof.enabled=true  --set ipv6.enabled=true  --set bpf.preallocateMaps=false  --set operator.image.tag=b0d3140aa079b31d2f3889b0f695cf7add393e68  --set hubble.listenAddress=:4244  --set etcd.leaseTTL=30s  --set image.useDigest=false  --set sessionAffinity=false  --set hubble.relay.image.repository=quay.io/cilium/hubble-relay-ci  --set ipv4.enabled=true  --set preflight.image.useDigest=false  --set operator.image.repository=quay.io/cilium/operator  --set debug.enabled=true  --set config.enabled=false   --set operator.enabled=false   --set agent=false   --set preflight.image.repository=quay.io/cilium/cilium-ci  --set operator.image.suffix=-ci  --set debug.verbose=flow  --set hubble.enabled=true  --set image.tag=b0d3140aa079b31d2f3889b0f695cf7add393e68  --set enableCnpStatusUpdates=true  --set preflight.enabled=true   --set nodeinit.enabled=false  --set preflight.image.tag=b0d3140aa079b31d2f3889b0f695cf7add393e68  --set image.repository=quay.io/cilium/cilium-ci  --set operator.image.useDigest=false  --set kubeProxyReplacement=disabled  --set ipam.operator.clusterPoolIPv6PodCIDR=fd02::/112  --set k8s.requireIPv4PodCIDR=true  --set logSystemLoad=true  --set nativeRoutingCIDR=10.0.0.0/8  --set bpf.masquerade=false  --set hubble.relay.image.useDigest=false  --set bandwidthManager=false  
To succeed, but it failed:
Exitcode: 1 
Err: exit status 1
Stdout:
 	 
Stderr:
 	 WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/jenkins/workspace/cilium-v1.11-k8s-1.19-kernel-4.9/src/github.com/cilium/cilium/test/vagrant-kubeconfig
	 WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/jenkins/workspace/cilium-v1.11-k8s-1.19-kernel-4.9/src/github.com/cilium/cilium/test/vagrant-kubeconfig
	 Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: Service "hubble-peer" in namespace "kube-system" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "cilium-preflight": current value is "cilium"
	 

/home/jenkins/workspace/cilium-v1.11-k8s-1.19-kernel-4.9/src/github.com/cilium/cilium/test/k8sT/Updates.go:132
```

### Standard Output

```shell
Number of "context deadline exceeded" in logs: 0
Number of "level=error" in logs: 0
Number of "level=warning" in logs: 0
Number of "Cilium API handler panicked" in logs: 0
Number of "Goroutine took lock for more than" in logs: 0
No errors/warnings found in logs
Number of "context deadline exceeded" in logs: 0
Number of "level=error" in logs: 1
Number of "level=warning" in logs: 1
Number of "Cilium API handler panicked" in logs: 0
Number of "Goroutine took lock for more than" in logs: 0
Top 2 errors/warnings:
Failed to update lock: Operation cannot be fulfilled on leases.coordination.k8s.io \
Unable to restart pod
Number of "context deadline exceeded" in logs: 0
Number of "level=error" in logs: 0
⚠️  Number of "level=warning" in logs: 12
Number of "Cilium API handler panicked" in logs: 0
Number of "Goroutine took lock for more than" in logs: 0
Top 5 errors/warnings:
Waiting for k8s node information
Failed to remove old router IPs (restored IP: 10.0.1.105) from cilium_host. Manual intervention is required to remove all other old IPs.
Failed to remove old router IPs (restored IP: fd02::13a) from cilium_host. Manual intervention is required to remove all other old IPs.
Failed to remove old router IPs (restored IP: 10.0.0.217) from cilium_host. Manual intervention is required to remove all other old IPs.
Failed to remove old router IPs (restored IP: fd02::c0) from cilium_host. Manual intervention is required to remove all other old IPs.
Cilium pods: [cilium-jkhmk cilium-sjlcc]
Netpols loaded: 
CiliumNetworkPolicies loaded: default::l7-policy 
Endpoint Policy Enforcement:
Pod                        Ingress   Egress
migrate-svc-server-fjp2k             
migrate-svc-server-jrdhv             
migrate-svc-server-pkcpf             
coredns-7c74c644b-dp997              
app1-786c6d794d-khttk                
app2-58757b7dd5-8z2fh                
migrate-svc-client-94p55             
migrate-svc-client-vp2k7             
migrate-svc-client-pwd4f             
app1-786c6d794d-nh55h                
app3-5d69599cdd-4f5vm                
migrate-svc-client-b5rlz             
migrate-svc-client-kmwqx             
Cilium agent 'cilium-jkhmk': Status: Ok  Health: Ok Nodes "" ContinerRuntime:  Kubernetes: Ok KVstore: Ok Controllers: Total 38 Failed 0
Cilium agent 'cilium-sjlcc': Status: Ok  Health: Ok Nodes "" ContinerRuntime:  Kubernetes: Ok KVstore: Ok Controllers: Total 50 Failed 0
```

### Standard Error

```shell
15:51:47 STEP: Running BeforeAll block for EntireTestsuite K8sUpdates
15:51:47 STEP: Ensuring the namespace kube-system exists
15:51:47 STEP: WaitforPods(namespace="kube-system", filter="-l k8s-app=cilium-test-logs")
15:51:47 STEP: WaitforPods(namespace="kube-system", filter="-l k8s-app=cilium-test-logs") => <nil>
15:52:06 STEP: Deleting Cilium and CoreDNS...
15:52:09 STEP: Waiting for pods to be terminated..
15:52:09 STEP: Cleaning Cilium state (b0d3140aa079b31d2f3889b0f695cf7add393e68)
15:52:09 STEP: Cleaning up Cilium components
15:52:10 STEP: Waiting for Cilium to become ready
15:52:22 STEP: Cleaning Cilium state (v1.10)
15:52:22 STEP: Cleaning up Cilium components
15:52:23 STEP: Waiting for Cilium to become ready
15:52:53 STEP: Deploying Cilium 1.10
15:52:53 STEP: Waiting for Cilium to become ready
15:53:49 STEP: Validating Cilium Installation
15:53:49 STEP: Performing Cilium controllers preflight check
15:53:49 STEP: Performing Cilium health check
15:53:49 STEP: Performing Cilium status preflight check
15:53:49 STEP: Checking whether host EP regenerated
15:53:50 STEP: Performing Cilium service preflight check
15:53:50 STEP: Performing K8s service preflight check
15:53:52 STEP: Waiting for cilium-operator to be ready
15:53:52 STEP: WaitforPods(namespace="kube-system", filter="-l name=cilium-operator")
15:53:52 STEP: WaitforPods(namespace="kube-system", filter="-l name=cilium-operator") => <nil>
15:53:52 STEP: Cilium "1.10" is installed and running
15:53:52 STEP: Restarting DNS Pods
15:54:07 STEP: Waiting for kube-dns to be ready
15:54:07 STEP: WaitforPods(namespace="kube-system", filter="-l k8s-app=kube-dns")
15:54:07 STEP: WaitforPods(namespace="kube-system", filter="-l k8s-app=kube-dns") => <nil>
15:54:07 STEP: Running kube-dns preflight check
15:54:08 STEP: Performing K8s service preflight check
15:54:08 STEP: Creating some endpoints and L7 policy
15:54:08 STEP: WaitforPods(namespace="default", filter="-l zgroup=testapp")
15:54:21 STEP: WaitforPods(namespace="default", filter="-l zgroup=testapp") => <nil>
15:54:25 STEP: Creating service and clients for migration
15:54:25 STEP: WaitforPods(namespace="default", filter="-l app=migrate-svc-server")
15:54:30 STEP: WaitforPods(namespace="default", filter="-l app=migrate-svc-server") => <nil>
15:54:30 STEP: WaitforPods(namespace="default", filter="-l app=migrate-svc-client")
15:54:35 STEP: WaitforPods(namespace="default", filter="-l app=migrate-svc-client") => <nil>
15:54:35 STEP: Validate that endpoints are ready before making any connection
15:54:36 STEP: Waiting for kube-dns to be ready
15:54:36 STEP: WaitforPods(namespace="kube-system", filter="-l k8s-app=kube-dns")
15:54:36 STEP: WaitforPods(namespace="kube-system", filter="-l k8s-app=kube-dns") => <nil>
15:54:36 STEP: Running kube-dns preflight check
15:54:37 STEP: Performing K8s service preflight check
15:54:38 STEP: Making L7 requests between endpoints
15:54:38 STEP: No interrupts in migrated svc flows
15:54:38 STEP: Install Cilium pre-flight check DaemonSet
FAIL: Timed out after 30.001s.
Unable to deploy preflight manifest
Expected command: helm install cilium-preflight /home/jenkins/workspace/cilium-v1.11-k8s-1.19-kernel-4.9/src/github.com/cilium/cilium/install/kubernetes/cilium --version=1.10.90 --namespace=kube-system  --set hubble.relay.image.tag=b0d3140aa079b31d2f3889b0f695cf7add393e68  --set hubble.eventBufferCapacity=65535  --set pprof.enabled=true  --set ipv6.enabled=true  --set bpf.preallocateMaps=false  --set operator.image.tag=b0d3140aa079b31d2f3889b0f695cf7add393e68  --set hubble.listenAddress=:4244  --set etcd.leaseTTL=30s  --set image.useDigest=false  --set sessionAffinity=false  --set hubble.relay.image.repository=quay.io/cilium/hubble-relay-ci  --set ipv4.enabled=true  --set preflight.image.useDigest=false  --set operator.image.repository=quay.io/cilium/operator  --set debug.enabled=true  --set config.enabled=false   --set operator.enabled=false   --set agent=false   --set preflight.image.repository=quay.io/cilium/cilium-ci  --set operator.image.suffix=-ci  --set debug.verbose=flow  --set hubble.enabled=true  --set image.tag=b0d3140aa079b31d2f3889b0f695cf7add393e68  --set enableCnpStatusUpdates=true  --set preflight.enabled=true   --set nodeinit.enabled=false  --set preflight.image.tag=b0d3140aa079b31d2f3889b0f695cf7add393e68  --set image.repository=quay.io/cilium/cilium-ci  --set operator.image.useDigest=false  --set kubeProxyReplacement=disabled  --set ipam.operator.clusterPoolIPv6PodCIDR=fd02::/112  --set k8s.requireIPv4PodCIDR=true  --set logSystemLoad=true  --set nativeRoutingCIDR=10.0.0.0/8  --set bpf.masquerade=false  --set hubble.relay.image.useDigest=false  --set bandwidthManager=false  
To succeed, but it failed:
Exitcode: 1 
Err: exit status 1
Stdout:
 	 
Stderr:
 	 WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/jenkins/workspace/cilium-v1.11-k8s-1.19-kernel-4.9/src/github.com/cilium/cilium/test/vagrant-kubeconfig
	 WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/jenkins/workspace/cilium-v1.11-k8s-1.19-kernel-4.9/src/github.com/cilium/cilium/test/vagrant-kubeconfig
	 Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: Service "hubble-peer" in namespace "kube-system" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "cilium-preflight": current value is "cilium"
	 

=== Test Finished at 2022-05-10T15:55:08Z====
15:55:08 STEP: Running JustAfterEach block for EntireTestsuite K8sUpdates
===================== TEST FAILED =====================
15:55:08 STEP: Running AfterFailed block for EntireTestsuite K8sUpdates
cmd: kubectl get pods -o wide --all-namespaces
Exitcode: 0 
Stdout:
 	 NAMESPACE           NAME                               READY   STATUS    RESTARTS   AGE     IP              NODE   NOMINATED NODE   READINESS GATES
	 cilium-monitoring   grafana-d69c97b9b-dwqfw            0/1     Running   0          73m     10.0.1.238      k8s2   <none>           <none>
	 cilium-monitoring   prometheus-655fb888d7-v62zx        1/1     Running   0          73m     10.0.1.205      k8s2   <none>           <none>
	 default             app1-786c6d794d-khttk              2/2     Running   0          62s     10.0.0.191      k8s1   <none>           <none>
	 default             app1-786c6d794d-nh55h              2/2     Running   0          62s     10.0.0.85       k8s1   <none>           <none>
	 default             app2-58757b7dd5-8z2fh              1/1     Running   0          62s     10.0.0.62       k8s1   <none>           <none>
	 default             app3-5d69599cdd-4f5vm              1/1     Running   0          62s     10.0.0.179      k8s1   <none>           <none>
	 default             migrate-svc-client-94p55           1/1     Running   0          40s     10.0.1.61       k8s2   <none>           <none>
	 default             migrate-svc-client-b5rlz           1/1     Running   0          40s     10.0.0.74       k8s1   <none>           <none>
	 default             migrate-svc-client-kmwqx           1/1     Running   0          40s     10.0.0.160      k8s1   <none>           <none>
	 default             migrate-svc-client-pwd4f           1/1     Running   0          40s     10.0.1.106      k8s2   <none>           <none>
	 default             migrate-svc-client-vp2k7           1/1     Running   0          40s     10.0.1.69       k8s2   <none>           <none>
	 default             migrate-svc-server-fjp2k           1/1     Running   0          45s     10.0.0.185      k8s1   <none>           <none>
	 default             migrate-svc-server-jrdhv           1/1     Running   0          45s     10.0.1.4        k8s2   <none>           <none>
	 default             migrate-svc-server-pkcpf           1/1     Running   0          45s     10.0.1.134      k8s2   <none>           <none>
	 kube-system         cilium-jkhmk                       1/1     Running   0          2m17s   192.168.56.12   k8s2   <none>           <none>
	 kube-system         cilium-operator-64b64bd755-dpvjr   1/1     Running   0          2m17s   192.168.56.11   k8s1   <none>           <none>
	 kube-system         cilium-operator-64b64bd755-pfg5b   1/1     Running   0          2m17s   192.168.56.12   k8s2   <none>           <none>
	 kube-system         cilium-sjlcc                       1/1     Running   0          2m17s   192.168.56.11   k8s1   <none>           <none>
	 kube-system         coredns-7c74c644b-dp997            1/1     Running   0          78s     10.0.0.219      k8s1   <none>           <none>
	 kube-system         etcd-k8s1                          1/1     Running   1          80m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         kube-apiserver-k8s1                1/1     Running   1          80m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         kube-controller-manager-k8s1       1/1     Running   2          80m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         kube-proxy-8smx7                   1/1     Running   0          75m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         kube-proxy-hqd85                   1/1     Running   0          74m     192.168.56.12   k8s2   <none>           <none>
	 kube-system         kube-scheduler-k8s1                1/1     Running   2          80m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         log-gatherer-26dgl                 1/1     Running   0          73m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         log-gatherer-vts2b                 1/1     Running   0          73m     192.168.56.12   k8s2   <none>           <none>
	 kube-system         registry-adder-d7cdq               1/1     Running   0          74m     192.168.56.11   k8s1   <none>           <none>
	 kube-system         registry-adder-l2x2g               1/1     Running   0          74m     192.168.56.12   k8s2   <none>           <none>
	 
Stderr:
 	 

Fetching command output from pods [cilium-jkhmk cilium-sjlcc]
cmd: kubectl exec -n kube-system cilium-jkhmk -c cilium-agent -- cilium endpoint list
Exitcode: 0 
Stdout:
 	 ENDPOINT   POLICY (ingress)   POLICY (egress)   IDENTITY   LABELS (source:key[=value])                       IPv6        IPv4         STATUS   
	            ENFORCEMENT        ENFORCEMENT                                                                                             
	 878        Disabled           Disabled          1204       k8s:app=migrate-svc-client                        fd02::1b7   10.0.1.69    ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                           
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                    
	                                                            k8s:io.kubernetes.pod.namespace=default                                            
	                                                            k8s:zgroup=migrate-svc                                                             
	 1078       Disabled           Disabled          1          k8s:cilium.io/ci-node=k8s2                                                 ready   
	                                                            reserved:host                                                                      
	 1378       Disabled           Disabled          14803      k8s:app=migrate-svc-server                        fd02::1a8   10.0.1.134   ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                           
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                    
	                                                            k8s:io.kubernetes.pod.namespace=default                                            
	                                                            k8s:zgroup=migrate-svc                                                             
	 2241       Disabled           Disabled          4          reserved:health                                   fd02::106   10.0.1.96    ready   
	 2714       Disabled           Disabled          1204       k8s:app=migrate-svc-client                        fd02::185   10.0.1.106   ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                           
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                    
	                                                            k8s:io.kubernetes.pod.namespace=default                                            
	                                                            k8s:zgroup=migrate-svc                                                             
	 3038       Disabled           Disabled          1204       k8s:app=migrate-svc-client                        fd02::146   10.0.1.61    ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                           
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                    
	                                                            k8s:io.kubernetes.pod.namespace=default                                            
	                                                            k8s:zgroup=migrate-svc                                                             
	 3278       Disabled           Disabled          14803      k8s:app=migrate-svc-server                        fd02::1dc   10.0.1.4     ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                           
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                    
	                                                            k8s:io.kubernetes.pod.namespace=default                                            
	                                                            k8s:zgroup=migrate-svc                                                             
	 
Stderr:
 	 

cmd: kubectl exec -n kube-system cilium-sjlcc -c cilium-agent -- cilium endpoint list
Exitcode: 0 
Stdout:
 	 ENDPOINT   POLICY (ingress)   POLICY (egress)   IDENTITY   LABELS (source:key[=value])                            IPv6       IPv4         STATUS   
	            ENFORCEMENT        ENFORCEMENT                                                                                                 
	 31         Disabled           Disabled          14803      k8s:app=migrate-svc-server                             fd02::f3   10.0.0.185   ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                        
	                                                            k8s:io.kubernetes.pod.namespace=default                                                
	                                                            k8s:zgroup=migrate-svc                                                                 
	 292        Disabled           Disabled          1204       k8s:app=migrate-svc-client                             fd02::2e   10.0.0.160   ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                        
	                                                            k8s:io.kubernetes.pod.namespace=default                                                
	                                                            k8s:zgroup=migrate-svc                                                                 
	 692        Enabled            Disabled          10924      k8s:id=app1                                            fd02::f5   10.0.0.191   ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=app1-account                                   
	                                                            k8s:io.kubernetes.pod.namespace=default                                                
	                                                            k8s:zgroup=testapp                                                                     
	 924        Disabled           Disabled          306        k8s:appSecond=true                                     fd02::e7   10.0.0.62    ready   
	                                                            k8s:id=app2                                                                            
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=app2-account                                   
	                                                            k8s:io.kubernetes.pod.namespace=default                                                
	                                                            k8s:zgroup=testapp                                                                     
	 1054       Disabled           Disabled          1          k8s:cilium.io/ci-node=k8s1                                                     ready   
	                                                            k8s:node-role.kubernetes.io/master                                                     
	                                                            reserved:host                                                                          
	 1399       Disabled           Disabled          5340       k8s:id=app3                                            fd02::66   10.0.0.179   ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                        
	                                                            k8s:io.kubernetes.pod.namespace=default                                                
	                                                            k8s:zgroup=testapp                                                                     
	 3135       Enabled            Disabled          10924      k8s:id=app1                                            fd02::c    10.0.0.85    ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=app1-account                                   
	                                                            k8s:io.kubernetes.pod.namespace=default                                                
	                                                            k8s:zgroup=testapp                                                                     
	 3842       Disabled           Disabled          1204       k8s:app=migrate-svc-client                             fd02::4d   10.0.0.74    ready   
	                                                            k8s:io.cilium.k8s.policy.cluster=default                                               
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=default                                        
	                                                            k8s:io.kubernetes.pod.namespace=default                                                
	                                                            k8s:zgroup=migrate-svc                                                                 
	 3950       Disabled           Disabled          61731      k8s:io.cilium.k8s.policy.cluster=default               fd02::8a   10.0.0.219   ready   
	                                                            k8s:io.cilium.k8s.policy.serviceaccount=coredns                                        
	                                                            k8s:io.kubernetes.pod.namespace=kube-system                                            
	                                                            k8s:k8s-app=kube-dns                                                                   
	 3963       Disabled           Disabled          4          reserved:health                                        fd02::3    10.0.0.44    ready   
	 
Stderr:
 	 

===================== Exiting AfterFailed =====================
15:55:19 STEP: Running AfterEach for block EntireTestsuite K8sUpdates
15:55:29 STEP: Cleaning up Cilium components
15:55:38 STEP: Waiting for Cilium to become ready
15:55:51 STEP: Running AfterEach for block EntireTestsuite

[[ATTACHMENT|7b6e257d_K8sUpdates_Tests_upgrade_and_downgrade_from_a_Cilium_stable_image_to_master.zip]]
15:55:53 STEP: Running AfterAll block for EntireTestsuite K8sUpdates
15:55:53 STEP: Cleaning up Cilium components
```

### Resources

- Jenkins URL: https://jenkins.cilium.io/view/Cilium-v1.11/job/cilium-v1.11-k8s-1.19-kernel-4.9/59/testReport/Suite-k8s-1/19/K8sUpdates_Tests_upgrade_and_downgrade_from_a_Cilium_stable_image_to_master/
- ZIP file(s): [7b6e257d_K8sUpdates_Tests_upgrade_and_downgrade_from_a_Cilium_stable_image_to_master.zip](https://github.com/cilium/cilium/files/8718223/7b6e257d_K8sUpdates_Tests_upgrade_and_downgrade_from_a_Cilium_stable_image_to_master.zip)

### Anything else?

A new issue exclusive to 1.11 for the `K8sUpdates Tests upgrade and downgrade from a Cilium stable image to master` test. I've added a link to an example but it happens in absolutely all 1.11 pipelines from K8s 1.16 to 1.21: https://jenkins.cilium.io/view/Cilium-v1.11/

Interestingly, this issue does not happen on 1.22 / 1.23.

It started to appear in pipelines on May 10, but was probably undetected since then because the same test was already failing beforehand (but with a different error). It looks like a Helm manifest malformation.

Digging into  backports merged to 1.11 around this time, it is likely the culprit is #19752, ~~with the backport of #19710 being a prime suspect.~~ the prime suspect is the backport of #18620.

**Comment by joestringer** [comment]
@nathanjsweet could this be related to 21e6e6ade37fb6ac9d38dd885c3f236bdb869c26 ?

**Comment by nbusseneau** [comment]
You're right, I think it's more related to this.

**Comment by nbusseneau** [comment]
I think this was fixed by f8d0e211687571a2e20853922471a70dc7bbc319.

cc @kaworu for confirmation.

**Comment by kaworu** [comment]
@nbusseneau correct, should we close this issue?
